src/templates/icon.overrides.liquid

```diff
@@ -1,3 +1,33 @@
 /*******************************
-         Theme Overrides
+Theme Overrides
 *******************************/
+
+/* Step 1: Common Duotone Properties: All required to make icons render reliably */
+i.icon.duotone {
```

Comment:
As all duotone icon names have a suffix, we could get rid of the additional classname (so "camera-duotone icon" is sufficient instead of "camera-duotone duotone icon"
```suggestion
i.icon[class^="-duotone"] {
```

Reply:
I don't get this one. When I use duotone icons, I use `camera duotone icon` and not `camera-duotone duotone icon`

Reply:
You added `-duotone` to each duotone icon as name
![image](https://user-images.githubusercontent.com/18379884/199983100-ed2987e7-87c7-40f9-9945-b9e382c0ed8e.png)

The getClassName function takes this name and creates classnames out of this. As a result `camera-duotone` is being generated. If we don't do this, then , yes, a separate `duotone` class makes sense

Reply:
Oh I see. You can guess I hacked duotone support without a deep understanding. I see that outline and thin icons define both the `-thin` and `.thin` way of using icons, and basically I copied that. Personnally I use camera duotone and not `camera-duotone`. I can remove the `-duotone` but it will make it different from how thin and outline icons work.

Reply:
I looked at the code again....infact it will NOT generate `-duotone`.. Sorry, it will, just as for outline and thin, generate a separte classname....all good, i will revoke my related review comments